terra-batch-libchart/templates/_wds.tpl

```diff
@@ -72,25 +59,3 @@ spec:
 {{- define "terra-batch-libchart.wds-service" -}}
 {{- include "terra-batch-libchart.util.merge" (append . "terra-batch-libchart.wds-service.tpl") -}}
 {{- end -}}
-
-
-{{- define "terra-batch-libchart.wds-config.tpl" -}}
```

Comment:
for other ConfigMaps, I think we are keeping the common elements in terra-batch-libchart even when those elements are pretty limited (e.g. see the ConfigMap for the reverse proxy).

I might suggest doing the same here -- i.e. keeping the definition, `apiVersion`, `kind`, and `metadata` here in the libchart, and overwriting the `data` in each of `coa-helm` and `cromwell-helm`